javascript/javascript3/week2/index.js

```diff
@@ -0,0 +1,62 @@
+BASE_URL = "https://open.er-api.com/v6/latest/USD";
+
+const baseCurrency = document.getElementById("baseCurrency");
+const targetCurrency = document.getElementById("targetCurrency");
+const btnSwapCurrency = document.getElementById("btnSwapCurrency");
+const btnConvertCurrency = document.getElementById("btnConvertCurrency");
+const amountToConvert = document.getElementById("amount");
+
+const currencyList = async () => {
+  try {
+    const response = await fetch(BASE_URL);
+    if (response.status !== 200) {
+      return `something went wrong ${response.status.toString()}!`;
+    }
+    const data = await response.json();
+    let eurValue;
+    let dkkValue;
+    Object.entries(data.rates).forEach(([key, value]) => {
+      if (key === "EUR") eurValue = value;
+      else if (key === "DKK") dkkValue = value;
+      const baseOption = document.createElement("option");
+      const targetOption = document.createElement("option");
+      baseOption.text = key;
+      baseOption.value = value;
+      targetOption.text = key;
+      targetOption.value = value;
+
+      baseCurrency.add(baseOption);
+      targetCurrency.add(targetOption);
+    });
+    baseCurrency.value = eurValue;
+    targetCurrency.value = dkkValue;
+    //console.log(eurValue);
+  } catch (error) {
+    return error.message;
+  }
+};
+
+window.onload = (event) => {
```

Comment:
Forgetting async await, though not necessary here. Otherwise, could just do `window.onload = currencyList`